### PR TITLE
Feature/151

### PR DESCRIPTION
**Pull request by Muhaddatha**
Implemented functionality to add resources to workspace from OSB repository.
Added styled tabs to dialog component.

feature151 1 0

### CODE REVIEW

File: applications/osb-portal/src/components/common/OSBDialog.tsx [hunk 1]
```diff
@@ -18,14 +18,34 @@ import "react-markdown-editor-lite/lib/index.css";
...
   actions?: React.ReactElement;
   closeAction: () => void;
 }
 
+const useStyles = makeStyles((theme) => ({
+  dialog: {
+    padding: 0,
+    "& .MuiDialogContent-root": {
+      padding: 0,
+      backgroundColor: '#3b3b3b',
```
Comment (filippomc) [inline applications/osb-portal/src/components/common/OSBDialog.tsx:38]: Better change to the closest color in the theme variables.

File: applications/osb-portal/src/components/workspace/AddResourceForm.tsx [hunk 2]
```diff
@@ -25,9 +57,297 @@ function isValidHttpUrl(s: string) {
...
+      {value === index && (
+        <Box>{children}</Box>
+      )}
+
+    </div>
+  );
+}
+
+
+const useStyles = makeStyles((theme) => ({
```
Comment (filippomc) [inline applications/osb-portal/src/components/workspace/AddResourceForm.tsx:80]: 200+ lines of styling for a single component are not per se wrong, but don't smell good. Better think at some way to reduce the amount of styling

File: applications/osb-portal/src/components/workspace/AddResourceForm.tsx [hunk 3]
```diff
@@ -68,51 +412,131 @@ export default (props: WorkspaceEditProps) => {
...
+        />
+        <Tab 
+          className={classes.tab}
+          label={<>
+            <Typography className={classes.tabTitle} component="span">From OSB Repository</Typography>
+            <LinkIcon />
+            </>}
+        />
+        <Tab
+          className={classes.tab}
```
Comment (filippomc) [inline applications/osb-portal/src/components/workspace/AddResourceForm.tsx:449]: This tab functionality is not implemented yet, so better remove it for now

**Review by filippomc — COMMENTED** [review]
Looks good, thanks!
A few comments: we need to remove the useless tab (for now) and it would be nice to reduce the amount of styling needed

File: applications/osb-portal/src/components/common/OSBDialog.tsx (hunk 1 above)
Comment (Muhaddatha) [reply applications/osb-portal/src/components/common/OSBDialog.tsx:38]: Ok

**Pull request merged**